app.js

```diff
@@ -1,4 +1,29 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+
+var express = require('express');
+var app = express();
+var trucks = {
```

Comment:
You want to use the trucks module to return the trucks for your application. All the data you need is in the foodTrucks array in trucks.js, and you want to use the getTrucks() or getTrucks(name) methods from that module to return data to your routes.